mantidproject/mantid
# D2B orientation fix

D2B is currently mirrored from the correct orientation. This issue should fix it, that is &theta -> -&theta;.

**To test:**

Run `LoadEmptyInstrument` and load the updated IDF. Check the orientation of the instrument is similar to that shown here https://www.ill.eu/instruments-support/instruments-groups/instruments/d2b/description/instrument-layout/.

Please also merge https://github.com/mantidproject/mantidgeometry/pull/98 at the same time.

Fixes #20406.

*Does not need to be in the release notes.* - while this changes the instrument view other data reduction should not be affected

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
